pool_test.go

```diff
@@ -692,6 +693,7 @@ func TestPoolStatSuccessfulAcquireCounters(t *testing.T) {
 	assert.Equal(t, int64(2), stat.AcquireCount())
 	assert.Equal(t, int64(1), stat.EmptyAcquireCount())
 	assert.True(t, stat.AcquireDuration() > lastAcquireDuration)
+	assert.True(t, stat.EmptyAcquireWaitTime() < stat.AcquireDuration())
```

Comment:
assert.Greater should be preferred to compare things. The error reported if it fails will be clearer

You should also change the one on previous line.

Reply:
I would like to recommend you using testifylint available via golangci-lint.

I can help if you create an issue for me, assign it me, and you are not in  a hurry 🤣